jobs/transforms/base_model_audits.py

```diff
@@ -251,6 +251,60 @@ def process(self, input_model):
             yield audit_errors.ModelDomainObjectValidateError(input_model, e)
 
 
+@audit_decorators.AuditsExisting(
+    base_models.BaseCommitLogEntryModel, base_models.BaseSnapshotMetadataModel)
+class ValidateCommitCmdsSchema(beam.DoFn):
+    """DoFn to validate schema of commit commands in commit_cmds dict.
+    """
+
+    def _get_change_domain_class(self, unused_item):
+        """Returns a Change domain class.
+
+        This should be implemented by subclasses.
+
+        Args:
+            unused_item: datastore_services.Model. Entity to validate.
+
+        Returns:
+            change_domain.BaseChange. A domain object class for the
+            changes made by commit commands of the model.
+
+        Raises:
+            NotImplementedError. This function has not yet been implemented.
+        """
+        raise NotImplementedError(
```

Comment:
@YBCS Add another validator which inherits this as a base class and but has the implementation of the _get_change_domain_class, for example for the exploration models.

Reply:
Done! @rohitkatlaa PTAL!

Reply:
@YBCS not Done. You have to include another Validator and not a test, for example

```python
@audit_decorators.AuditsExisting(exp_models.ExplorationSnapshotMetadataModel)
class ValidateExplorationSnapshotMetadataModelCommitCmdsSchema(base_model_audits.ValidateCommitCmdsSchema)):
```

Please add it in the appropriate file and add the tests for this. Thanks.

Reply:
@rohitkatlaa Hello, I have made some changes, PTAL!